File: src/net/eduard/permissions/command/PermissionsGroupAddPermissionCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.permissions.EduPermissions;
import net.eduard.permissions.manager.PermissionsGroup;
import net.eduard.permissions.manager.PermissionsManager;

public class PermissionsGroupAddPermissionCommand extends CommandManager {

	public PermissionsGroupAddPermissionCommand() {
		super("addpermission", "adicionarpermissao");
		setUsage("/permissions group addpermission <group> <permission>");
		setDescription("Adicionar permissao para o grupo");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

		PermissionsManager manager = EduPermissions.getInstance().getManager();
		if (args.length < 4) {
			sendUsage(sender);
		} else {
			String nome = args[2];
			PermissionsGroup group = manager.getGroup(nome);
			String perm = args[3];
			if (group != null) {
				group.getPermissions().add(perm);
				sender.sendMessage(EduPermissions.getInstance().message("group-add-permission")
						.replace("$group", "" + nome).replace("$permission", perm));
			} else {
				sender.sendMessage(
						EduPermissions.getInstance().message("group-not-exists").replace("$group", "" + nome));
			}
//			
		}

		return true;
	}
}

File: src/main/java/net/eduard/permissions/command/PermissionsGroupAddPermissionCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.Sender

import net.eduard.permissions.api.PermissionsAPI
import net.eduard.permissions.core.PermsMessages

class PermissionsGroupAddPermissionCommand :
    Command("addpermission", "adicionarpermissao") {


    override fun onCommand(sender: Sender, args: List<String>) {
        val manager = PermissionsAPI.instance
        if (args.size < 4) {
            sendUsage(sender)
            return
        }
        val nome = args[2]
        val group = manager.getGroup(nome)
        val perm = args[3]
        if (group != null) {
            group.addPermission(perm,!perm.startsWith("-"))
            sender.sendMessage(
                PermsMessages.message("group-add-permission")
                    .replace("\$group", "" + nome)
                    .replace("\$permission", perm)
            )
            return
        }
        sender.sendMessage(
            PermsMessages.message("group-not-exists")
                .replace("\$group", "" + nome)
        )


        //

    }


    init {
        usage = "/permissions group addpermission <group> <permission>"
        description = "Adicionar permissao para o grupo"
    }
}